_templates/sidebar_links.html

```diff
@@ -38,6 +38,18 @@
 
 </div>
 
+<div class="sidebarblock">
+  <h3>{{ _('For developers') }}</h3>
+  <ul class="simple">
+    <li><a class="reference external" href="https://gitter.im/ipython/ipython">
+            Development Chat Room</a></li>
```

Comment:
we said we wouldn't promote gitter room too much. Can we put mailing list first if you want a non-github communication channel.